client/src/components/layout/DestinationShow.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+
+const DestinationShow = (props) => {
+  const [destination, setDestination] = useState({
+    id: "",
+    name: "",
+    type: "",
+    location: "",
+    description: "",
+    website: "",
+  });
+  const { id, name, type, location, description, website } = destination;
+  const getDestination = async () => {
+    const destinationId = props.match.params.id;
+    try {
+      const response = await fetch(`/api/v1/destinations/${destinationId}`);
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`;
+        const error = new Error(errorMessage);
+        throw error;
+      }
+      const body = await response.json();
+      const destination = body.destination;
+      setDestination(destination);
+    } catch (err) {
+      console.error(`Error in fetch ${err.message}`);
+    }
+  };
+  useEffect(() => {
+    getDestination();
+  }, []);
+  return (
+    <>
+      <div className="destination">
+        <h1>{name}</h1>
+        <p>{description}</p>
+        <ul>
+          <li>{type}</li>
+          <li>{location}</li>
+          <li>{website}</li>
+        </ul>
+      </div>
+
+      <div className="reviews">
+        <h1>Reviews:</h1>
+      </div>
```

Comment:
we should not include placeholder or filler text of where things will go in the future - this is called future proofing often leads to unorganized code where we have to remember to go back and change things. Instead, only add the features and functionality that this feature needs and is using right now